inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -459,6 +459,8 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
                 res->params["operation"] = "max";
             } else if (node->description() == "Minimum") {
                 res->params["operation"] = "min";
+            } else if (node->description() == "Abs") {
```

Comment:
What specific creator do you use? this creator or creator for `Abs` operation?

Reply:
I use creator for Abs operation
But it seems to break something for gna plugin
Trying to fix the failures